obspy/io/nordic/core.py

```diff
@@ -973,6 +1290,124 @@ def _write_nordic(event, filename, userid='OBSP', evtype='L', outdir='.',
         return
 
 
+def _write_moment_tensor_line(focal_mechanism):
+    """
+    Generate the two lines required for moment tensor solutions in Nordic.
+    """
+    # First line contains hypocenter info, second contains tensor info
+    lines = [list(' ' * 79 + 'M'), list(' ' * 79 + 'M')]
```

Comment:
I've not seen this before but its actually kind of a nice way to get a mutable string :smile: